pkg/cmd/root_test.go

```diff
@@ -76,9 +77,17 @@ func TestPluginList(t *testing.T) {
 	err = ioutil.WriteFile(nd.Join("tkn-nonexec"), []byte("nonexec"), 0o600)
 	assert.NilError(t, err)
 	defer env.Patch(t, "PATH", nd.Path()+":/non/existing/path")()
+
+	// Reset the TKN_PLUGINS_DIR so that during local test
+	// existing plugins are not considered using tests
+	pluginHome := os.Getenv("TKN_PLUGINS_DIR")
+	os.Setenv("TKN_PLUGINS_DIR", "/non/existing/path")
```

Comment:
this would be buggy for other tests, if you don't use a defer env.Patch to cleanup the setting